config/webpack.vendor.js

```diff
@@ -1 +1,121 @@
-module.exports = ['react-intl', 'intl'];
+/* eslint-disable */
+
+module.exports = [
+  'intl',
+  /**
+   * React related
+   */
+  '@apollo/client',
+  '@apollo/client/link/batch-http',
+  '@reduxjs/toolkit',
+  'core-js/shim',
+  'regenerator-runtime/runtime',
+  'formik',
+  'rc-input-number',
+  'react',
+  'react-ace',
+  'react-dom',
+  'react-dnd',
+  'react-dnd-html5-backend',
+  'react-debounce-input',
+  'react-diff-view',
+  'react-ellipsis-with-tooltip',
+  'react-onclickoutside',
+  'react-password-strength',
+  'react-router-dom',
+  'react-router-bootstrap',
+  'react-loading-skeleton',
+  'react-redux',
+  'redux',
+  'redux-logger',
+  'redux-thunk',
+  'reselect',
+  'prop-types',
+  'classnames',
+  'seamless-immutable',
+  'connected-react-router',
+  'react-helmet',
+  'react-intl',
+
+  /**
+   * Patternfly related
+   */
+  'patternfly-react',
+  'patternfly-react-extensions',
+  '@patternfly/react-core',
+  '@patternfly/react-icons',
+  '@patternfly/react-table',
+  '@patternfly/react-tokens',
+  '@patternfly/react-styles',
+  '@patternfly/react-charts',
+  // '@redhat-cloud-services/frontend-components',
```

Comment:
remove from here to rh_cloud https://github.com/theforeman/foreman_rh_cloud/issues/973